frontend/src/pages/DashboardPage/Dashboard/Dashboard.jsx

```diff
@@ -580,8 +591,24 @@ const Dashboard = () => {
   }, [graphData]);
 
   return (
-    <div className="dashboard-container">
-      {/* <div className="text-container">
+      <>
```

Comment:
If need be, can we add the loading only for the graph div so that the rest of the page is rendered while only the graph is loading? Just wanted to check if that's a possibility.